adr/00003-color-tokens-spec.md

```diff
@@ -0,0 +1,43 @@
+# 00003 - Color tokens specification
+
+# Status
+
+Decided
+
+# Context
+
+Since we develop our design system, there is a need to have tokens for colors
+with clearly defined relationships between them, so the user will be able to use them accordingly to their purpose.
+
+# Decision
+
+#### 1. We use the following structure:
+
+`--mu-[type]-[element]-[role]-[variant]-[state?]`
+
+where:
+
+- `type` is color
+- `element` is one of: button | button-text
+- `role` is one of: default | critical | info | success | warning | accent
+- `variant` is one of primary | secondary | tertiary
```

Comment:
**Question:** shouldn't it be layer? like so: `--mu-[type]-[element]-[role]-[layer]-[state?]` (design system tokens)

and for semantic components: `--mu-[type]-[element]-[component]-[role]-[variant]`